e2e-tests/playwright/tests/functional/channels/settings/display/theme_accessibility.spec.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Since this is mainly accessibility test, I suggest to move this to `/e2e-tests/playwright/test/specs/accessibility`. So that eventually, we can easily run Accessibility tests in a separate test group.